lib/takeout.rb

```diff
@@ -0,0 +1,61 @@
+class Takeout
+
+  def initialize
+    @menu = Menu.new
+    @order = Order.new
+    @message = Message.new
+    @delivery = Delivery.new
```

Comment:
I would inject these as parameters to avoid strong coupling between this Service class and these smaller classes.

It would make test isolation easier, too.